### PR TITLE
Add bells to policies when a new version is available

### PR DESCRIPTION
**Pull request by RubenRBS**
This adds some bells to the policies dropdown when a new version of a policy is available.
It was initially made to adress #2464, but currently it still shows the privacy policy on first view, which does not fix the linked issue.

Ideas welcomed :)

### CODE REVIEW

File: static/main.js [hunk 1]
```diff
@@ -277,14 +277,29 @@ function initializeResetLayoutLink() {
...
     jsCookie.remove('fs_uid');
     jsCookie.remove('cookieconsent_status');
-    if (options.policies.privacy.enabled &&
-        options.policies.privacy.hash !== jsCookie.get(options.policies.privacy.key)) {
-        $('#privacy').trigger('click', {
-            title: 'New Privacy Policy. Please take a moment to read it',
-        });
+    if (options.policies.privacy.enabled) {
+        if (jsCookie.get(options.policies.privacy.key) == null) {
+            $('#privacy').trigger('click', {
```
Comment (mattgodbolt) [inline static/main.js:284]: super nit, here you use `$(selector)` below you use `$(document).find(selector)`. We've always used the former before, just wondering if there's an advantage either way? If not, consistency would be ace :)

File: static/main.js [hunk 2]
```diff
@@ -299,11 +314,24 @@ function initPolicies(options) {
...
     // '' means no consent. Hash match means consent of old. Null means new user!
     var storedCookieConsent = jsCookie.get(options.policies.cookies.key);
-    if (options.policies.cookies.enabled && storedCookieConsent !== '' &&
-        options.policies.cookies.hash !== storedCookieConsent) {
-        simpleCooks.show();
-    } else if (options.policies.cookies.enabled && hasCookieConsented(options)) {
-        analytics.initialise();
+    if (options.policies.cookies.enabled) {
+        if (storedCookieConsent !== '' && options.policies.cookies.hash !== storedCookieConsent) {
+            simpleCooks.show();
```
Comment (mattgodbolt) [inline static/main.js:319]: Nice - so there's a wee notification saying there's something to read. Certainly less intrusive... will check out the branch and see what I think. I like the idea, thanks Rubén.

**Review by mattgodbolt — COMMENTED** [review]
I like it, will be back after I try it out :)

File: static/main.js (hunk 1 above)
Comment (partouf) [reply static/main.js:284]: makes sense, id's should be unique, so .find should be irrelevant

**Comment by mattgodbolt** [comment]
Thanks Ruben!  Took me a while to remember I had to enable the policies locally to see them!

Seems like if there's no cookies set locally it pops up as before? I think that means it doesn't cover #2464 I think, as the OP there suffers from it due to no prior settings.

My experience locally in a new incognito

* Privacy policy popped up on startup
* Cookie consent  footer appearer
* Hit "close" to close PP
* Noticed the nice bell in policies (love the UX there)
* Still had the "consent/no consent" cookie thing at the bottom, along with "Read the new policy"
* Clicked Consent.
* Policy bell still there
* Clicked policy bell
* There was a bell next to "cookie policy", which went away when I read it

This is probably as intended - but maybe we shouldn't pop up the loud privacy policy thing over the top any more? Hmm, I don't know. Thoughts? Maybe what I saw wasn't quite as intended?

File: static/main.js (hunk 1 above)
Comment (RubenRBS) [reply static/main.js:284]: Good catch :)

**Comment by RubenRBS** [comment]
Thanks for the review Matt. Yeah, as you noticed, the modal still shows if the site thinks that it's your first visit, which happens when we can't find the privacy cookie, which makes it so that the linked issue is not resolved by this, you're right.

I left the pane because I'm still not sure how the legal stuff works there (As I think we have it so it blocks all the page for a reason) - But then again, I don't see many websites filling the screen with their privacy policy before using them, so we might be able to not show like this!

As for the Consent not removing the bells, you're right, that's an oversight, I'll get it fixed soon :)

**Comment by mattgodbolt** [comment]
Nice! This is a strict improvement I think. I saw a tweet today poking fun at "updated Privacy Policy" things^, so ... this is timely :) 

^ not aimed at us!

**Review by mattgodbolt — APPROVED** [review]
Looks great: feel free to update whenever. This means we can push out the typo fix #2432 without feeling _too_ guilty about popping up on lots of people.

**Comment by mattgodbolt** [comment]
er, s/update/merge/. my brain...

**Comment by partouf** [comment]
might as well click the button then

**Pull request merged**